cluster/gce/configure-vm.sh

```diff
@@ -559,11 +559,8 @@ if [[ -z "${is_push}" ]]; then
   set-kube-env
   [[ "${KUBERNETES_MASTER}" == "true" ]] && mount-master-pd
   create-salt-pillar
-  if [[ "${KUBERNETES_MASTER}" == "true" ]]; then
-    create-salt-master-auth
```

Comment:
> Would it be useful to add the salt-set-apiserver call to this

That will work as long as $MASTER_NAME isn't set to the external IP of the master (as @zmerlynn mentioned). 

I'm hoping that @erictune will have time today to look at #6949 now that it passes e2e tests.